cirq-web/bloch_sphere_example.ipynb

```diff
@@ -33,28 +33,7 @@
    "id": "fbf7b67d",
    "metadata": {},
    "source": [
-    "### Cirq-web visualization"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "e4bf9c7d",
-   "metadata": {},
-   "source": [
-    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
-    "  <td>\n",
```

Comment:
These links are still useful even in Github view. Let's remove the open on Quantum AI button. Also the download can point to the Github raw URL. Also we should point to this ipynb file not the circuits ipynb (copy pasta! 🍝 )